runelite-client/src/main/java/net/runelite/client/plugins/pyramidplunder/PyramidPlunderConfig.java

```diff
@@ -0,0 +1,154 @@
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.pyramidplunder;
+import java.awt.Color;
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("pyramidplunder")
+public interface PyramidPlunderConfig extends Config
+{
+	@ConfigItem(
+		position = 0,
+		keyName = "hideTimer",
+		name = "Hide default timer",
+		description = "Hides the default pyramid plunder timer"
+	)
+	default boolean hideTimer()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 1,
+		keyName = "showExactTimer",
+		name = "Show exact timer",
+		description = "Displays the amount of time remaining as an infobox"
+	)
+	default boolean showExactTimer()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "timerLowWarning",
+		name = "Timer low warning",
+		description = "Determines the time when the timers color will change"
+	)
+	default int timerLowWarning()
+	{
+		return 30;
+	}
+
+	@ConfigItem(
+		position = 3,
+		keyName = "highlightDoorsColor",
+		name = "Highlight doors color",
+		description = "Selects the color for highlighting tomb doors"
+	)
+	default Color highlightDoorsColor()
+	{
+		return Color.green;
+	}
+
+	@ConfigItem(
+		position = 4,
+		keyName = "highlightDoors",
+		name = "Highlight doors",
+		description = "Highlights the four tomb doors in each room"
+	)
+	default boolean highlightDoors()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 5,
+		keyName = "highlightSpeartrapColor",
+		name = "Highlight speartrap color",
+		description = "Selects the color for highlighting speartraps"
+	)
+	default Color highlightSpeartrapsColor()
+	{
+		return Color.orange;
+	}
+
+	@ConfigItem(
+		position = 6,
+		keyName = "highlightSpeartraps",
+		name = "Highlight speartraps",
+		description = "Highlight the spear traps at the entrance of each room"
+	)
+	default boolean highlightSpeartraps()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 7,
+		keyName = "highlightContainersColor",
+		name = "Highlight containers color",
+		description = "Selects the color for highlighting urns, chests and sarcophagus"
+	)
+	default Color highlightContainersColor()
+	{
+		return Color.yellow;
+	}
+
+	@ConfigItem(
+		position = 8,
+		keyName = "highlightUrnsFloor",
+		name = "Highlight urns floor",
+		description = "Highlight the urns starting at selected floor and up"
+	)
+	default int highlightUrnsFloor()
+	{
+		return 9;
```

Comment:
Defaulting these to 9 seems too restricted for the standard player?

Reply:
My thinking is that it fills the screen up with the overlay so the standard player would not want it on, it also had the added benefit of not trying to figure out how the average player plays. I think efficient starts at being able to loot floor 8 and uses the following:
- urns starting floor 7
- chest starting floor 6
- sarcophagus starting never (9)

What numbers do you think I should use?

Reply:
Oh, right. There is only 8 floors. That seems fine to use 9 as off then by default.